packages/browser-repl/src/components/types/show-dbs-output.tsx

```diff
@@ -12,14 +12,19 @@ type DatabaseObject = {
   map: any;
 };
 
+function formatBytes(value: number): string {
+  const precision = value <= 1000 ? '0' : '0.0';
+  return numeral(value).format(precision + 'b');
```

Comment:
```suggestion
  const precision = value <= 1000 ? '0' : '0.00';
  return numeral(value).format(precision + ' ib');
```

would make this closer to what we want/had previously and actually use 1024-based units, I think